src/app/attendance/actions/getAttendance.ts

```diff
@@ -0,0 +1,30 @@
+"use server";
+
+import { Tables } from "@/lib/db";
+import { getSupabaseClient } from "../../../../utils/supabase/supabaseClient";
+import { getMembers } from "@/app/members/actions/getMembers";
+
+export const getAttendance = async () => {
+  const supabase = await getSupabaseClient();
+
+  const membersResponse = await getMembers();
+  const members = membersResponse?.data || [];
+
+  const { data: attendanceData } = await supabase
+    .from(Tables.Attendance)
+    .select("*");
+
+  const enrichedData = (attendanceData || []).map((attendance) => {
```

Comment:
@Mubashar1009  I think there should be a better name and why we need to use map here?